README.md

```diff
@@ -32,7 +32,27 @@ To run this app, you will need to generate a secure 256-bit keys in hex format t
 - To run development mode, make sure the docker-compose file has volume and tsnd command has --poll, otherwise it won't reload on changes
 - Docker compose volume for the app must be set to only src folder, not the root folder. Using root folder will make the dockerized app unable to run cause of invalid executable's binary (sharp package) to run (since node modules will contain executables for windows, not for linux)
 
-## TODOs
+## Jobs
 
-- endpoints to queue the OG jobs
-- bull can probably be updated to use bull-mq for native typescript support
+There are 2 main jobs that will be done by puppeteer server:
+
+### Yacht Scoring Jobs
+
+`Puppeteer Server` main objective is to test the credentials to external services, such as `YachtScoring` that the users to imported into their SYRF account. The process is as follow:
+
+1. User submit the credentials via the SYRF website
+2. SYRF Website will send the credentials to the SYRF Backend API (`Live Data Server`)
+3. `Live Data Server` will encrypt the credentials and queue it to redis, for `Puppeteer Server` to pick up.
+4. `Puppeteer Server` will launch a puppeteer browser pointing to `YachtScoring` (or other external services in the future) and fill in the Login form.
+5. If login with the credentials is successful, `Puppeteer Server` will store the encrypted credentials to the database, linked to the SYRF user that submits the request. The user can then import an event from yachtscoring into SYRF.
+
+To import an existing event from `YachtScoring`:
+
+1. User submit request to get the events from their account in `YachtScoring` through SYRF Website Event management page.
+2. SYRF Website will submit the request to the SYRF Backend API
+3. `Live Data Server` will queue a job to redis for `Puppeteer Server` to pick up
+4. `Puppeteer Sserver` will launch a puppeteer browser, login to `YachtScoring` using stored credentials, and head to the event list page on `YachtScoring`
+5. The data from this event list page will be returned from the Job, and added to the response to the SYRF Website.
+6. SYRF Website will show the list of events available for the user to choose from
+7. Once the user select one of the event, `Live Data Server` will queue another job for `Puppeteer Server` to start scraping the vessel and participant list to copy into the SYRF event.
+8. Once `Puppeteer Server` done scraping, it will send the details to another queue that will be processed by `Live Data Server` to validate and process into the event in SYRF
```

Comment:
3rd job is to create screenshots for opengraph

Reply:
Yeah, I'm adding that as well, forgot to continue when encountered the bug with yachtscoring. 
Actually there's only 2 job being run in puppeteer, the `importingEventJob` was just for adding to the queue. The job itself is running in Live Data Server